### PR TITLE
Test statistics

### PR DESCRIPTION
**Pull request by bearney74**
fix 2 out of 3 TODO python tests in test_statistics.py

### CODE REVIEW

File: Lib/test/test_statistics.py [hunk 1]
```diff
@@ -2914,10 +2912,8 @@ def setUp(self):
     def tearDown(self):
         sys.modules['statistics'] = statistics
 
-    # TODO: RUSTPYTHON, ValueError: math domain error
-    @unittest.expectedFailure
-    def test_inv_cdf(self): # TODO: RUSTPYTHON, remove when this passes
-        super().test_inv_cdf() # TODO: RUSTPYTHON, remove when this passes
+    def test_inv_cdf(self):
+        super().test_inv_cdf()
```
Comment (fanninpm) [inline Lib/test/test_statistics.py:2916]: The `# TODO: RUSTPYTHON, remove when this passes` comments mean that this should be removed **completely**.

Comment (youknowone) [inline Lib/test/test_statistics.py:2916]:
```suggestion
```

@fanninpm making a suggestion will be easier

**Review by youknowone — CHANGES_REQUESTED** [review]
Looks good. could you check @fanninpm's comment and the CI failure? it is formatting error. You can fix it using `cargo fmt --all`

File: stdlib/src/math.rs [hunk 2]
```diff
@@ -211,7 +211,9 @@ mod math {
     #[pyfunction]
     fn sqrt(value: ArgIntoFloat, vm: &VirtualMachine) -> PyResult<f64> {
         let value = *value;
-        if value.is_nan() { return Ok(value); }   //return nan if value is nan
+        if value.is_nan() {
+            return Ok(value);
+        } //return nan if value is nan
```
Comment (DimitrisJim) [inline stdlib/src/math.rs:216]: Don't think this really requires a comment here, can just remove it:

```suggestion
        if value.is_nan() {
            return Ok(value);
        }
```

**Comment by fanninpm** [comment]
Also, I think @youknowone prefers _rebasing_ to using merge commits.

**Review by youknowone — APPROVED** [review]
Thank you for contributing!

@fanninpm When I can keep the commit history, rebase is preferred. In this case, patch is small enough to squash. So it is ok.

**Pull request merged**